app/portainer/__module.js

```diff
@@ -385,6 +385,17 @@ angular
         },
       };
 
+      var initTimeout = {
+        name: 'portainer.init.timeout',
```

Comment:
I don't see a reason why we need to put it under the init views. just `admin-timeout` should be ok (or an external HTML file, like I suggested above)

Reply:
fixed